### PR TITLE
Remove wildcards from 1.8 InfluxQL schema doc

### PR DESCRIPTION
**Pull request by pierwill**
See https://github.com/influxdata/docs-v2/issues/3201#issuecomment-992927665.

### CODE REVIEW

File: content/influxdb/v1.8/query_language/explore-schema.md [hunk 1]
```diff
@@ -438,10 +438,7 @@ See the Data Exploration page for documentation on the
 [`OFFSET` clause](/influxdb/v1.8/query_language/explore-data/#the-offset-clause),
 and on [Regular expressions in queries](/influxdb/v1.8/query_language/explore-data/#regular-expressions).
 
-The following queries with wildcards characters (`*`) are also allowed:
-
-- `SHOW MEASUREMENTS ON *.*` to show all databases and all RPs
-- `SHOW MEASUREMENTS ON <db>.*` to show a specific database and all its RPs
+Run `SHOW MEASUREMENTS ON <db>.*` to show a specific database and all its RPs.
```
Comment (davidby-influx) [inline content/influxdb/v1.8/query_language/explore-schema.md:441]: I think this line has to go as well.  @lesam ?

**Review by davidby-influx — CHANGES_REQUESTED** [review]
I believe all the wildcards have to be removed.  @lesam ?  Please confirm.

**Review by sanderson — COMMENTED** [review]
@davidby-influx @lesam Is this feature supported in Enterprise 1.9? What about in the current 2.x and Cloud InfluxQL implementations?

**Comment by davidby-influx** [comment]
@sanderson - I don't think it is in 1.9 Enterprise. It is slated for 1.10 Enterprise.

No idea about cloud. 

This is in 2.1.0 OSS

**Comment by lesam** [comment]
@sanderson not yet supported in cloud, the issue is here: https://github.com/influxdata/idpe/issues/11806

File: content/influxdb/v1.8/query_language/explore-schema.md (hunk 1 above)
Comment (lesam) [reply content/influxdb/v1.8/query_language/explore-schema.md:441]: Yes. Previous behaviour, preserved in all versions:

```
show measurements # only valid from CLI after use database, same as:
show measurements on <database> # shows all measurements from all retention policies, not broken out by policy
```

Additional behaviour in OSS 2.1.0 and Enterprise 1.10:
```
show measurements on *.* # all databases and all retention policies, broken out by retention policy
show measurements on <db>.* # specific database and all RPs, broken out by retention policy (different than non-wildcard which is not broken by retention policy, for backward-compatibility)
show measurements on <db>.<rp> # specific database and specific RP
```

Error everywhere:
```
show measurements .*
show measurements on .*
```

**Comment by sanderson** [comment]
@lwandzura Although we're removing this wildcard support from the 1.8 InfluxQL docs, we need to make sure it is added to the InfluxQL docs in OSS 2.x and Cloud.

**Review by sanderson — APPROVED** [review]

**Pull request merged**